dockerfiles/theia/Dockerfile

```diff
@@ -77,52 +77,13 @@ RUN yarn
 # Run into production mode
 RUN che:theia production
 
+# Compile plugins
+RUN cd plugins && ./foreach_yarn
```

Comment:
Sorry, @sun where is located script `foreach_yarn` or maybe you forgot to push it? I can not find it.

Reply:
@AndrienkoAleksandr  it is provided by the generator when you perform a che:theia init